### PR TITLE
COVID-19 query generator

### PR DESCRIPTION
**Pull request by lintool**
Query generator specifically for COVID-19. If it finds a variant of COVID-19 in the query, replaces it with all the variants of COVID-19, including 2019-nCOV and SARS-COV-2.

### CODE REVIEW

File: src/test/java/io/anserini/search/query/BagOfWordsQueryGeneratorTest.java [hunk 1]
```diff
@@ -29,7 +30,7 @@
 public class BagOfWordsQueryGeneratorTest {
   @Test
   public void test1() {
-    Analyzer analyzer = new EnglishAnalyzer();
+    Analyzer analyzer = IndexCollection.DEFAULT_ANALYZER;
```
Comment (rodrigonogueira4) [inline src/test/java/io/anserini/search/query/BagOfWordsQueryGeneratorTest.java:33]: What will be the default analyzer? Just curious, why don't we use the English analyzer?

Comment (lintool) [reply src/test/java/io/anserini/search/query/BagOfWordsQueryGeneratorTest.java:33]: This is a recent change to standardize the default analyzer... basically, we _are_ using the `EnglishAnalyzer()`

File: src/main/java/io/anserini/search/query/Covid19QueryGenerator.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    BooleanQuery.Builder builder = new BooleanQuery.Builder();
+    for (String t : tokens) {
+      builder.add(new TermQuery(new Term(field, t)), BooleanClause.Occur.SHOULD);
+    }
+
+    QueryParser parser = new QueryParser(IndexArgs.CONTENTS, analyzer);
+
+    try {
+      List<Query> disjuncts = new ArrayList<>();
+      disjuncts.add(parser.parse("\"COVID-19\""));
```
Comment (rodrigonogueira4) [inline src/main/java/io/anserini/search/query/Covid19QueryGenerator.java:63]: Is there a way to verify that these new terms will improve (and not hurt) effectiveness without having any annotation?

Comment (lintool) [reply src/main/java/io/anserini/search/query/Covid19QueryGenerator.java:63]: Nothing is sure without judgments... ;)

But: https://lucene.apache.org/core/8_0_0/core/org/apache/lucene/search/DisjunctionMaxQuery.html

File: src/test/java/io/anserini/search/query/BagOfWordsQueryGeneratorTest.java (hunk 1 above)
Comment (rodrigonogueira4) [reply src/test/java/io/anserini/search/query/BagOfWordsQueryGeneratorTest.java:33]: Oh ok, thanks!

File: src/main/java/io/anserini/search/query/Covid19QueryGenerator.java (hunk 2 above)
Comment (rodrigonogueira4) [reply src/main/java/io/anserini/search/query/Covid19QueryGenerator.java:63]: Ok!

**Pull request merged**